### ISSUE TITLE
PostgreSQL: T4 generates invalid code if table/view with same name exists in different schemas

### ISSUE THREAD

**Issue by zer** [linq2db/linq2db#1191]
I tested the T4 data model generation with the sample PostgreSQL database from here: https://github.com/NorfolkDataSci/adventure-works-postgres

Unfortunately the generated code is invalid because it contains classes with the same name and/or properties in the same class with the same name. 
The result are for example the following errors (just two of many):
> CS0102	The type 'AdventureworksDB' already contains a definition for 'Crs'
> CS0102	The type 'Sr' already contains a definition for 'Modifieddate'

See this gist for the generated model: https://gist.github.com/zer/209c2eaa2077654ff9bfbf9daac36cd2

### Steps to reproduce
Generate data model from the database from here: https://github.com/NorfolkDataSci/adventure-works-postgres

OR 

the short version:
1. Create new Database
2. Create schemas and tables:
`CREATE SCHEMA s1;`
`CREATE SCHEMA s2;`
`CREATE TABLE s1.t1 (c1 int);`
`CREATE TABLE s2.t1 (c1 int);`
3. Generate data model from the newly created database
4. Result should look something like this: https://gist.github.com/zer/209c2eaa2077654ff9bfbf9daac36cd2#file-simple-models-generated-cs

### Environment details
linq2db version: *2.0.0*  
linq2db.t4models: *1.10.0*
Database Server: *PostgreSQL 10.4*  
Database Provider: *Npgsql 3.2.5*  
Operating system: *Windows 10*  
Framework version: *.NET Core 2.0*

**Comment by MaceWindu** [comment]
Working on fix for next release. For now you can try to use 
```cs
GenerateSchemaAsType = true;
```
to move non-default schema tables to separate class
